src/components/flat-table/flat-table.pw.tsx

```diff
@@ -0,0 +1,3619 @@
+  FlatTableNoAccSubRowComponent,
+  FlatTableAccSubRowComponent,
+  FlatTableFirstColExpandableComponent,
+  FlatTableAlreadyExpandedComponent,
+  FlatTableExpandAllComponent,
+  FlatTableAllSubrowSelectableComponent,
+  FlatTableParentSubrowSelectableComponent,
+  FlatTableChildSubrowSelectableComponent,
+  FlatTablePartiallySelectedOrHighlightedRows,
+  KeyboardNavigationWithPagination,
+  HighlightedRowWithLoadingState,
+  FlatTableDraggableComponent,
+  FlatTablePagerStickyHeaderComponent,
+  FlatTableCheckboxComponent,
+  FlatTableFirstColumnHasRowspan,
+  FlatTableLastColumnHasRowspan,
+} from "./components.test-pw";
+import { FlatTableProps } from "./flat-table.component";
+import Icon from "../icon";
+import { getDataElementByValue } from "../../../playwright/components";
+import {
+  batchSelectionCounter,
+  batchSelectionButtonsByPosition,
+} from "../../../playwright/components/batch-selection/index";
+import {
+  actionPopover,
+  actionPopoverButton,
+} from "../../../playwright/components/action-popover/index";
+import { relLink } from "../../../playwright/components/link/index";
+import {
+  flatTable,
+  flatTableWrapper,
+  flatTableHeader,
+  flatTableHeaderCells,
+  flatTableHeaderRowByPosition,
+  flatTableBody,
+  flatTableCell,
+  flatTableCheckboxHeader,
+  flatTableHeaderCellsIcon,
+  flatTableExpandableIcon,
+  flatTableBodyRows,
+  flatTableRowHeader,
+  flatTableBodyRowByPosition,
+  flatTableSortable,
+  flatTableSubrows,
+  flatTableSubrowByPosition,
+  flatTableDraggableItem,
+  flatTableDraggableItemByPosition,
+  flatTableCaption,
+  flatTablePager,
+  flatTablePageSizeSelect,
+  flatTablePageSelectListPosition,
+  pageSelectInput,
+  flatTablePageSelectNext,
+  flatTablePageSelectPrevious,
+  flatTableCurrentPageInput,
+  flatTableCheckboxCell,
+  flatTableCheckboxAsProp,
+} from "../../../playwright/components/flat-table";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import {
+  checkCSSOutline,
+  assertCssValueIsApproximately,
+  getRotationAngle,
+  checkAccessibility,
+  getStyle,
+  waitForAnimationEnd,
+} from "../../../playwright/support/helper";
+import { HooksConfig } from "../../../playwright";
+
+const sizes = [
+  ["compact", "8px", "13px", 24],
+  ["small", "16px", "14px", 32],
+  ["medium", "16px", "14px", 40],
+  ["large", "16px", "16px", 48],
+  ["extraLarge", "16px", "16px", 64],
+];
+
+const borderSizeSmall = "1px";
+const borderSizeMedium = "2px";
+const borderSizeLarge = "4px";
+
+const heightWidth = [150, 250, 600, 1000];
+
+const colorThemes = [
+  ["dark", "rgb(51, 91, 112)", "rgb(102, 132, 148)"],
+  ["light", "rgb(204, 214, 219)", "rgb(179, 194, 201)"],
+  ["transparent-base", "rgb(242, 245, 246)", "rgb(242, 245, 246)"],
+  ["transparent-white", "rgb(255, 255, 255)", "rgb(255, 255, 255)"],
+];
+
+const gold = "rgb(255, 188, 25)";
+const black = "rgba(0, 0, 0, 0.9)";
+const lightGold = "rgb(255, 188, 26)";
+const greyBlack = "rgb(0, 0, 0)";
```

Comment:
Is this rgb code correct? (0, 0, 0) is black. Is it missing a percentage may be to make it greyBlack? Could it possibly be transposed with the black colour above?